test/scss/_master.test.scss

```diff
@@ -0,0 +1,44 @@
+// to change the brand in the testing environement, we need to reset the variables
+$o-colors-is-silent: true !global;
+
+$_o-colors-brand: if(global-variable-exists(o-brand), if($o-brand != null, $o-brand, 'master'), 'master');
+
+$o-colors-palette: () !global;
+$o-colors-tints: () !global;
+
+$_o-colors-experimental-palette: (
+	'slate-white-15':        #dedfe0, // oColorsMix('slate', 'white', 15): to support work on internal brand design
+	'slate-white-5':         #f4f4f5, // oColorsMix('slate', 'white', 5): to support work on internal brand design
+);
+
+$_o-colors-experimental-palette-warning: true !global;
```

Comment:
Nice to have: Now the pallet is branded 🎉, we can default to the warning being `false` and only add the experimental pallet if the brand is "internal". That way master brand users won't get the annoying warning :D 

_edit: oops commenting on a test, same applies to the actual codebase_